### DISCUSSION TITLE
API Release — Means You Can...

### DISCUSSION THREAD

**Discussion by Conza88** [bisq-network/bisq#5237] [Ideas]
Now do what? 👍 What would you like to see be created? What could we see?

Mobile app's that allow you to make and take offer's?

**Comment by BtcContributor** [comment]
I would say:
- Mobile app as you said
- Support for raspberry so that a bisq daemon could always be online and use the API with some external service

**Comment by Conza88** [comment]
**Mobile app:**
- Make offers
- Take offers
- Edit offers
- Be alerted by new offers at certain price points, margin difference, payment methods, local fiat [combination of all]
- Progress trade
- Chat with trading partner

**Other:**
- Trade & transaction history data e.g. linked to Specter? Specter Warden?
- Whirlpool?

**Reply by Conza88** [reply]
Respond to mediations etc.
